Source/WebCore/platform/mediastream/mac/AVVideoCaptureSource.mm

```diff
@@ -795,18 +795,24 @@ static bool isZoomSupported(const Vector<VideoPreset>& presets)
     setSessionSizeFrameRateAndZoom();
 }
 
-static bool isSameFrameRateRange(AVFrameRateRange* a, AVFrameRateRange* b)
+static bool isFrameRateMatching(double frameRate, AVCaptureDevice* device)
 {
-    return a.minFrameRate == b.minFrameRate && a.maxFrameRate == b.maxFrameRate && !PAL::CMTimeCompare(a.minFrameDuration, b.minFrameDuration) && !PAL::CMTimeCompare(a.maxFrameDuration, b.maxFrameDuration);
+    auto frameDuration = PAL::CMTimeMake(1, frameRate);
```

Comment:
Nit: you can define this just before it is used

Reply:
done